src/Lookup/RestrictedEntityLookup.php

```diff
@@ -105,6 +105,15 @@ public function getEntityAccessCount() {
 	}
 
 	/**
+	 * Resets the number of entities loaded via this object.
+	 *
+	 * @since 3.2.1
```

Comment:
I'm not sure what you mean with "having to locally apply patches to the build for minimal updates".

> and re-tagging a minimal 3.3.0 release also sounds awry to me.

semver does not allow retagging in any siatuation, not even when you accidentally have a breaking change in a minor release. I'm not sure why you'd need to retag...